Sources/VitaminSwiftUI/Foundations/Assets/README.md

```diff
@@ -21,6 +21,12 @@ struct AssetsView: View {
 }
 ```
 
+
+For the flags, you can also use the specific API with alpha-2 country code (ISO 3166-1).
+```swift
+let frenchFlag: UIImage? = VitaminAssets.flag(countryCode: "fr")?.swiftUIImage
```

Comment:
I think there is an error due to a too quick copy/paste from UIKit README, no ? :)

```suggestion
let frenchFlag: Image? = VitaminAssets.flag(countryCode: "fr")?.swiftUIImage
```

Reply:
Sorry for that 🙈